contrib/debian/tests/ci

```diff
@@ -2,9 +2,9 @@
 set -e
 # try loading the mtdram module to run our mtd tests
 modprobe mtdram 2>&1 || true
-sed "s,^DisabledPlugins=.*,DisabledPlugins=," -i /etc/fwupd/fwupd.conf
-sed "s,^VerboseDomains=.*,VerboseDomains=*,"  -i /etc/fwupd/fwupd.conf
-sed "s,^AllowEmulation=false,AllowEmulation=true," -i /etc/fwupd/fwupd.conf
+fwupdtool modify-config DisabledPlugins ""
```

Comment:
Great idea. What about assuming "" if the value is not set? Or is that too magic?

Reply:
I think it's too magical, all other times it needs 3 arguments but if it's this one only 2 is ok?

Reply:
I could see us making a case that an empty argument "resets it to default", but then that doesn't help us - the default is `test;test_ble`.

Reply:
Right; agree - ignore my first suggestion :)